core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -707,7 +707,11 @@ protected void addOperationHandler(final Class<? extends Operation> opClass, fin
     }
 
     public <OP extends Output<O>, O> void addOperationHandler(final Class<? extends Output<O>> opClass, final OutputOperationHandler<OP, O> handler) {
-        operationHandlers.put(opClass, handler);
+        if (null == handler) {
+            operationHandlers.remove(opClass);
```

Comment:
addOperationHandler(oldOp,null) is this now a way to remove a previous assignment of an oldOp?